pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -123,6 +123,9 @@ func (m *Stub) Start() error {
 		defer m.wg.Done()
 		m.server.Serve(sock)
 	}()
+
+	<-time.After(5 * time.Second)
```

Comment:
rather than a hard wait that makes every test that calls this take five seconds longer, can we PollImmediate on the dial attempt until it succeeds (with a backstop timeout that returns the dial error encountered)?

Reply:
Thank you for the review. Fixed!